src/identity.js

```diff
@@ -536,6 +563,11 @@ export class Identity extends EventEmitter {
                 return this.popup;
             }
         }
+        // for safari in a non-popup context, remember that we've got a login in progress so we can
+        // work around some ITP issues when we come back from Schibsted Account
```

Comment:
Right... So I guess for the popup-case, we don't know what to do yet, maybe? Actually, I think I don't see why we can't set this in both cases...

Reply:
Done